conjureup/controllers/newcloud/common.py

```diff
@@ -52,9 +52,9 @@ def try_get_creds(cloud):
         return existing_creds['credentials'][cloud]['default-credential']
 
     # XXX we should really prompt to select because this is non-deterministic
-    for k in existing_creds['credentials'][cloud].keys():
+    for k, v in existing_creds['credentials'][cloud].items():
         if 'default-region' in k:
-            continue
+            app.current_region = v
```

Comment:
Not perfect but should be addressed in a seperate PR concerning credential handling